src/inject/planner_helper.js

```diff
@@ -193,10 +193,9 @@ PlannerHelper.prototype.createElement = function () {
 PlannerHelper.prototype.abort = function () {
     this.aborted = true;
 
-    var message = `
-        Sorry, it appears that either WebReg or one of the data dependencies for Planner Helper
-        have changed their format. An error report has been sent and an update will be released
-        soon to make Planner Helper compatible with these changes.`;
+    var message = "Sorry, it appears that either WebReg or one of the data dependencies for Planner Helper" +
```

Comment:
There needs to be a space at the end of this string or else the output will look like "...Planner Helperhave changed...". (same goes for line 197)